docs/en/latest/plugins/saml-auth.md

```diff
@@ -0,0 +1,123 @@
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## Description
+
+The `saml-auth` Plugin can be used to access SAML (Security Assertion Markup Language 2.0) IdP (Identity Provider)
+to do authentication, from the SP (service provider) perspective.
+
+## Attributes
+
+| Name      | Type | Required      | Description |
+| ----------- | ----------- | ----------- | ----------- |
+| `sp_issuer`      | string       | True      | SP name to access IdP.       |
+| `idp_uri`      | string       | True      | URI of IdP.       |
+| `idp_cert`      | string       | True      | IdP Certificate, used to verify saml response.       |
+| `login_callback_uri`      | string       | True      | redirect uri used to callback the SP from IdP after login.       |
+| `logout_uri`      | string       | True      | logout uri to trigger logout.       |
+| `logout_callback_uri`      | string       | True      | redirect uri used to callback the SP from IdP after logout.       |
+| `logout_redirect_uri`      | string       | True      | redirect uri after successful logout.       |
+| `sp_cert`      | string       | True      | SP Certificate, used to sign the saml request.       |
+| `sp_private_key`      | string       | True      | SP private key.       |
+
+## Enabling the Plugin
+
+You can enable the Plugin on a specific Route as shown below:
+
+```shell
+curl http://127.0.0.1:9180/apisix/admin/routes/test_saml_auth -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+{
+    "methods": ["GET"],
+    "uri": "/anything/*",
+    "plugins": {
+          "saml-auth": {
+            "idp_cert": "-----BEGIN CERTIFICATE-----\nMIIClzCCAX8CBgGC7zvjWjANBgkqhkiG9w0BAQsFADAPMQ0wCwYDVQQDDAR0ZXN0\nMB4XDTIyMDgzMDE0NDkxNVoXDTMyMDgzMDE0NTA1NVowDzENMAsGA1UEAwwEdGVz\ndDCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAJelXTy4j6pP++7Bq9Yp\nmkuDYoBQRhUUO/RNVNOBPitVisNEQgdpKS2CFYLENZePo0xsDrN+5IB/8MTEqo3V\ntVu2BYQtq5tS0SUEgH1TLIt6gcnOuqgmmk3+RfUM3WGaUlbEN/j1C50RepFJm03i\nAM8dWCsCTCnZQLq/T4T7wwEIIOuAfz7Zip9+8HuVTOUoC3SxwVqTNb0pVRvdg7gU\nYh0f4CnckhlZG+4AgFvyW5LXxyUWbBtIpYEJN8DZEbEg+QmjuUHT0w+Apdtr6MSM\nKT87tQMxRlIYCqjmwPB9xjW1eZ9FdTGu81WeIhvs3iwTBoH6qcBR9ZJltY/VXEhg\neGECAwEAATANBgkqhkiG9w0BAQsFAAOCAQEAhbVRy4EoJzr/B84g9V6aAobUHoEF\ntv02G2GS/3UxrcaBXMLUl/tJlgti+PwQGDzIEXP0b0LLdAh/5LZFeDFVbphXUk26\nXxj3T2ej9ZcsRbaF+yHi0iisyMNdx9gbosDYg8yx9+XWImQMe6+T6+7fjYnilWE8\n0nYUuxyf1GF20EwQ0sjcJbxuq/K/2DZfOT8eo8a5c1oJFB20rE+KEpZnhBnDTQ+s\nUXPjN0AYTCKtPiAd7D190Pk1CQUbfAUmeII9WstCJhmGqIeiSpHsa9mAa50GVT3B\nnHm5obfM+sBEmy78MnC+s9Kt/EquakxeEi+YZEVfgyORq+LwNaef5MuedA==\n-----END CERTIFICATE-----",
```

Comment:
we provide an example, it is enough, which is clear:

```json
 "idp_cert": "-----BEGIN CERTIFICATE----... ...-----END CERTIFICATE-----",
```

Reply:
Yes, let's do the same for other cert/key in this file